src/mmw/apps/modeling/tasks.py

```diff
@@ -146,8 +146,19 @@ def analyze_pointsource(area_of_interest):
     return [point_source_pollution(area_of_interest)]
 
 
+def ensure_aoi_4d(area_of_interest):
+    # In rare instances, this method receives a five-dimensional array for these
+    # coordinates. This check is here to verify whether the `area_of_interest` is
+    # four- or five-dimensional, then assigns `pairs` appropriately.
+    if type(area_of_interest['coordinates'][0][0][0][0]) is list:
```

Comment:
Ah, good point. It looked like the AOIs from instances of this kind of Rollbar failure were all `MultiPolygon` shapes, so I agree that this would be a better guard. The one I tested was a single ring, but presented as a `MultiPolygon`, so I'll go back and look at the other Rollbar failures to see if the others are sometimes/often/always single ring `MultiPolygon`s, too.